tests/test_flows.py

```diff
@@ -466,6 +479,9 @@ def test_client_capability_query(self):
         self.check_log(env.controller,
                        r"Result Code= SUCCESS, client MAC= {}, BSSID= {}"
                        .format(sta2.mac, env.agents[0].radios[0].mac))
+               
+        for line in association_frame.splitlines():
+            self.check_log(env.controller, r"{}".format(line))
```

Comment:
Can you use the start_line feature that was recently added to check_log to make sure that you're actually matching the right part of the log? The previous check_log call just before the loop will give you the starting line.

Reply:
Do you mean instead of the loop or just before that? I don't think I understood the purpose of this method.

Reply:
Never mind, I'll just rework the startline functionality entirely at some point.